setup.py

```diff
@@ -6,22 +6,24 @@
     long_description = fh.read()
 
 setuptools.setup(
-    name="acm_pca_cert_generator",
-    version="0.0.0",
-    author="Matt Burgess",
-    author_email="matthewburgess@digital.uc.dwp.gov.uk",
-    description="Generate certs using ACM PCA",
+    name="acm_cert_helper",
+    version="0.4.0",
```

Comment:
Let's talk about this :)  It was kept at "0.0.0" because of my struggles getting Circle to bump versions correctly.  I think I have a plan of attack for that now, but needs a wider team disussion and an agreement on a standardized approach.

Reply:
Ahh - The circle CI tests failed when it was any other value! Do they pass if it's "0.0.0"?